microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Optimize website speed using Lighthouse
 
-This tutorial teaches you how to use DevTools to find ways to make your websites load faster.
+This tutorial teaches you how to use Lighthouse, and other tools, in DevTools to find ways to make your websites load faster.
 
-<!-- terminology hit counts as of 1/26/2022:
-audit: 93 hits
-lighthouse: 9 -->
-This tutorial uses the **Lighthouse** tool.  The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft is not responsible for and has no control over the content of these sites and any data that may be collected.
-
-The **Lighthouse** tool was previously called the **Audits** tool.  "Audits panel" means the same thing as "Lighthouse tool".
+The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft isn't responsible for and has no control over the content of these sites and any data that may be collected.
```

Comment:
```suggestion
The **Lighthouse** tool provides links to content that's hosted on third-party websites.  Microsoft isn't responsible for and has no control over the content of these sites and any data that may be collected.
```
balancing efficient terseness w/ clarity for ease of parsing, this change might be a better balance